servoTest1.ino

```diff
@@ -0,0 +1,20 @@
+#include <Servo.h> //Servo Library
+
+Servo servoVarName; // servo object
+int servoPin(2); 
+
+void setup() {
+  servoVarName.attach(servoPin);
+}
+
+void loop() {
+  for (int pos =0; pos < 180; pos++) {
+    servoVarName.write(pos);
+    delay(20);
+  }
+  for (int pos = 180; pos > 0; pos-= 180/8) {
```

Comment:
I know this is a little picky, but could you fix the whitespace here?
`pos-= 180/8` should be `pos -= 180 / 8`

Reply:
Okay, thanks!